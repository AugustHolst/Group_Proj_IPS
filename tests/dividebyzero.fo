fun int main() =
    let zero = 0 in
    let d = 2 in
    d/zero