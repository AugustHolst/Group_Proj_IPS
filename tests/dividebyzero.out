Error in /home/aske/ips/af/GroupProject/fasto/bin/../tests/dividebyzero.asm line 30: Runtime exception at 0x00400048: break instruction executed; no code given.

Processing terminated due to errors.

